app/views/contexts/email_form_context.py

```diff
@@ -3,6 +3,7 @@
 
 def build_confirmation_email_form_context(email_confirmation_form):
     context = {
+        "show_sign_out_warning": True,
         "hide_signout_button": False,
```

Comment:
Minor but seems like we should use either `sign_out` or `signout` for both these vars

Reply:
since we already use "sign_out_url" I will change it to `sign_out`